SU2_CFD/src/variables/CNEMOEulerVariable.cpp

```diff
@@ -141,39 +141,48 @@ CNEMOEulerVariable::CNEMOEulerVariable(su2double val_pressure,
 
   /* Under-relaxation parameter. */
   LocalCFL.resize(nPoint) = su2double(0.0);
+
+  bool interp = config->GetSolutionInterpolation();

```

Comment:
Can we keep interp out until it will be fully integrated into the code?

Reply:
@WallyMaier this PR is to feature_NEMO, so that's not an issue at the moment